bower.json

```diff
@@ -34,7 +34,6 @@
     "jquery-pjax": "1.9.6",
     "jquery-validation": "1.15.1",
     "magnific-popup": "https://github.com/wet-boew/Magnific-Popup.git#1.0.0+keyboard_trap_fix",
-    "openlayers": "https://github.com/wet-boew/openlayers-dist.git#2.13.1",
```

Comment:
There's currently a merge conflict on this line. PR #8101 added a line directly before it (see https://github.com/wet-boew/wet-boew/pull/8101/files#diff-0a08a7565aba4405282251491979bb6b).

To fix the conflict, this PR will need to be rebased again and the conflict will need to be manually resolved. I could be mistaken, but I don't believe any extra options can be used with ``git rebase`` to automatically resolve the conflict while retaining both changes.